src/extra/gfx/gltf.cpp

```diff
@@ -19,13 +20,9 @@ struct GLTFShard {
   static SHTypesInfo inputTypes() { return CoreInfo::AnyTableType; }
   static SHTypesInfo outputTypes() { return Types::DrawableHierarchy; }
 
-  static SHParametersInfo parameters() {
-    static Parameters params{
-        {"Transform", SHCCSTR("The transform variable to use"), {TransformVarType}},
-        {"Path", SHCCSTR("The static path to load a model from during warmup"), {CoreInfo::StringType}},
-    };
-    return params;
-  }
+  PARAM_PARAMVAR(_transformVar, "Transform", "The transform variable to use", {TransformVarType});
+  PARAM_VAR(_staticModelPath, "Path", "The static path to load a model from during warmup", {CoreInfo::StringType});
```

Comment:
This used to be a simple `std::string` and is now a `Var`. What is the impact? SHGouldn't we have a macro to still allow the direct usage of `std::string`? I think they were performance reason for doing so. @sinkingsugar

Reply:
Not sure I get it, where did we use std::string?

Reply:
Oh, ParamVar clones so Var is good

Reply:
> Not sure I get it, where did we use std::string?

About everywhere in `imgui.cpp`

Reply:
Yeah I got that after, but as I said its cloning so fine